commands/run/config/csv_templates/dna_const_single.csv

```diff
@@ -0,0 +1,2 @@
+clarity_sample_id,id,type,sex,assay,diagnosis,phenotype,group,father,mother,clarity_pool_id,platform,read1,read2,analysis,priority
```

Comment:
New CSV template system, to accomodate the differences both intra SMD and SMD <-> nf-core